File: ProAgil.Domain/model/Palestrante.cs
using ProAgil.Domain.Model;
using System;
using System.Collections.Generic;
using System.Text;

namespace ProAgil.Domain.Model
{
   public  class Palestrante
    {
        public int Id { get; set; }
        public string Nome { get; set; }

        public string MiniCurriculo { get; set; }

        public string ImagemUrl { get; set; }

        public string Telefone { get; set; }

        public string Email { get; set; }

        public List<RedeSocial> Redesociais { get; set; }

        public List<PalestrateEvento> PalestrateEventos { get; set; }
    }
}

File: ProAgil-App/src/app/_models/Palestrante.ts
import { Lote } from './Lote';
import { RedeSocial } from './RedeSocial';
import { Evento } from './Evento';

export interface Palestrante {
    id: number;
    nome: string;

    miniCurriculo: string;

    imagemUrl: string;

    telefone: string;

    email: string;

    redesociais: RedeSocial[];

    palestrateEventos: Evento[];
}
